.gitignore

```diff
@@ -12,3 +12,16 @@
 *.out
 *.gz
 *.toc
+
+# Build files #
+###############
+
+Makefile
+*.o
+moc_*
```

Comment:
Do we wish to ignore `moc` or `.moc` files?

Reply:
Currently `moc_` files are created and placed in a *moc* folder which has all files within ignored. There are no `.moc files from what I can see, so this is correct.